src/AutoLoader.php

```diff
@@ -1,30 +1,33 @@
 <?php declare(strict_types=1);
-require_once ('Support/CollectionProxy.php');
-require_once ('Support/Arr.php');
-require_once ('Support/Collection.php');
-require_once ('Support/Helpers.php');
-require_once ('Support/HigherOrderCollectionProxy.php');
-require_once ('Support/Str.php');
-require_once ('Helper/RequestError.php');
-require_once ('Helper/MyParcelCollection.php');
-require_once ('Helper/MyParcelCurl.php');
-require_once ('Helper/SplitStreet.php');
-require_once ('Helper/LabelHelper.php');
-require_once ('Helper/CheckoutFields.php');
-require_once ('Services/CheckApiKeyService.php');
-require_once ('Services/ConsignmentEncode.php');
-require_once ('Services/CollectionEncode.php');
-require_once ('Concerns/HasCheckoutFields.php');
-require_once ('Model/MyParcelRequest.php');
-require_once ('Adapter/ConsignmentAdapter.php');
-require_once ('Model/MyParcelConsignment.php');
-require_once ('Model/Consignment/AbstractConsignment.php');
-require_once ('Model/Consignment/BpostConsignment.php');
-require_once ('Model/Consignment/DPDConsignment.php');
-require_once ('Model/Consignment/PostNLConsignment.php');
-require_once ('Model/MyParcelCustomsItem.php');
-require_once ('Model/FullStreet.php');
-require_once ('Factory/ConsignmentFactory.php');
+require_once('Support/CollectionProxy.php');
+require_once('Support/Arr.php');
+require_once('Support/Collection.php');
+require_once('Support/Helpers.php');
+require_once('Support/HigherOrderCollectionProxy.php');
+require_once('Support/Str.php');
+require_once('Helper/RequestError.php');
+require_once('Helper/MyParcelCollection.php');
+require_once('Helper/MyParcelCurl.php');
+require_once('Helper/SplitStreet.php');
+require_once('Helper/LabelHelper.php');
+require_once('Helper/CheckoutFields.php');
+require_once('Services/CheckApiKeyService.php');
+require_once('Services/ConsignmentEncode.php');
+require_once('Services/CollectionEncode.php');
+require_once('Concerns/HasCheckoutFields.php');
+require_once('Model/MyParcelRequest.php');
+require_once('Adapter/ConsignmentAdapter.php');
+require_once('Model/MyParcelConsignment.php');
+require_once('Model/DeliveryOptions/DeliveryOptions.php');
+require_once('Model/DeliveryOptions/PickupLocation.php');
+require_once('Model/DeliveryOptions/ShipmentOptions.php');
```

Comment:
DeliveryOptions, PickupLocation en ShipmentOptions zijn bij nader inzien echte adapters (sorry dat ik het eerder een model noemde). Moet dus in /Adapter.

Je kan dan ook een DeliveryOptionsV2, PickupLocationV2 en ShipmentOptionsV2 maken om oude gegevens uit de order te kunnen verwerken en DeliveryOptionsV3, PickupLocationV3 en ShipmentOptionsV3 om de nieuwe gegevens te verwerken.
De gedeelde methods zou je dan in een abstract class kunnen plaatsen. Dan krijg je in DeliveryOptionsV3:
`class DeliveryOptionsV3 extends AbstractDeliveryOptionsAdapter`
en in AbstractDeliveryOptionsAdapter:
`abstract class AbstractDeliveryOptionsAdapter` (met de gedeelde methods uit DeliveryOptionsV3 en DeliveryOptionsV2)

Reply:
Done

Reply:
Done